Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- build.gradle	(date 1511239361000)
+++ build.gradle	(revision )
@@ -3,9 +3,10 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.0.1'
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
@@ -15,6 +16,7 @@
 allprojects {
     repositories {
         jcenter()
+        google()
     }
 }
 
Index: app/src/main/java/necowneco/tablayout/AddArt.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/necowneco/tablayout/AddArt.java	(date 1511239361000)
+++ app/src/main/java/necowneco/tablayout/AddArt.java	(revision )
@@ -9,7 +9,11 @@
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
 import android.widget.Spinner;
+import android.widget.TextView;
+import android.widget.Toast;
 
 import com.example.a888888888.sport.R;
 
@@ -67,19 +71,32 @@
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+
         View view = inflater.inflate(R.layout.fragment_add_art, null);
         view.setOnTouchListener(this);
         Spinner spinner=(Spinner)view.findViewById(R.id.SportClassSpinner);
-        final String[] SportList = {"雞腿飯", "魯肉飯", "排骨飯", "水餃", "陽春麵"};
-        ArrayAdapter<String> sportlist = new ArrayAdapter<String>(this.getContext(),
+        Button submitArt=(Button)view.findViewById(R.id.inputart);
+        EditText ct=(EditText)view.findViewById(R.id.ConTitle);
+        EditText cc=(EditText)view.findViewById(R.id.Concon);
+        final TextView test = (TextView)view.findViewById(R.id.testtext);
+        final String[] SportList = {"所有運動","有氧運動","走路","跑步","伏地挺身","仰臥起坐"};
+        ArrayAdapter<String> sportlist = new ArrayAdapter<String>(view.getContext(),
                 android.R.layout.simple_spinner_dropdown_item,
                 SportList);
         spinner.setAdapter(sportlist);
+        submitArt.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                test.setText("按下送出");
+                Toast.makeText(getContext(),test.getText(), Toast.LENGTH_SHORT).show();
+            }
+        });
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_add_art, container, false);
+        return view;
     }
 
     // TODO: Rename method, update argument and hook method into UI event
+
     public void onButtonPressed(String Tag,String number) {
         if (mListener != null) {
             mListener.onFragmentInteraction(Tag,number);
@@ -122,4 +139,5 @@
         // TODO: Update argument type and name
         void onFragmentInteraction(String Tag,String number);
     }
+
 }
Index: app/src/main/res/layout/fragment_add_art.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/fragment_add_art.xml	(date 1511239361000)
+++ app/src/main/res/layout/fragment_add_art.xml	(revision )
@@ -14,13 +14,13 @@
 
         <TextView
             android:id="@+id/textView11"
-            android:layout_width="match_parent"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="貼文標題：" />
 
         <EditText
-            android:id="@+id/editText"
-            android:layout_width="match_parent"
+            android:id="@+id/ConTitle"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:ems="10"
             android:inputType="textPersonName"
@@ -28,34 +28,40 @@
 
         <TextView
             android:id="@+id/textView17"
-            android:layout_width="match_parent"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="運動類型" />
 
         <Spinner
             android:id="@+id/SportClassSpinner"
-            android:layout_width="match_parent"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content" />
 
         <TextView
             android:id="@+id/textView18"
-            android:layout_width="match_parent"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="貼文內容：" />
 
         <EditText
-            android:id="@+id/editText2"
-            android:layout_width="match_parent"
+            android:id="@+id/Concon"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:ems="10"
             android:inputType="textPersonName"
             tools:text="請輸入內容" />
 
         <Button
-            android:id="@+id/button18"
-            android:layout_width="match_parent"
+            android:id="@+id/inputart"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="送出" />
+
+        <TextView
+            android:id="@+id/testtext"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="測試(失敗)" />
     </LinearLayout>
 
 </FrameLayout>
